.github/workflows/pinot_tests.yml

```diff
@@ -333,7 +333,7 @@ jobs:
       matrix:
         test_suite: [ "compatibility-verifier/multi-stage-query-engine-test-suite" ]
         old_commit: [
-          "release-1.2.0", "master"
+          "release-1.3.0", "release-1.2.0", "master"
```

Comment:
(minor) Let's keep the order from old to new

Reply:
Updated.